packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,144 @@
+
+## `ChipGroup` API
+
+| Prop              | Type                                                             | Default     | Description                                                                                                                    | Required |
+| ----------------- | ---------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **children**      | `React.ReactNode`                                                | `undefined` | Accepts multiple `Chip` components                                                                                             | ✅       |
+| **defaultValue**  | `string \|string[]`                                              | `undefined` | Sets the initial value of the Chip group                                                                                       | ❌       |
+| **isDisabled**    | `boolean`                                                        | `false`     | Disables or enables ChipGroup, it will propagate down to all the Chips                                                         | ❌       |
+| **name**          | `string`                                                         | `undefined` | The name of the chip group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name) | ❌       |
+| **onChange**      | `({ name, values }: { name: string; values: string[] }) => void` | `undefined` | Calls a function on any state change within the ChipGroup                                                                      | ❌       |
+| **selectionType** | `'single' \| 'multiple'`                                         | `'single'`  | Defines the selection behavior within the ChipGroup component, simulating radio/checkbox like behavior within the group        | ❌       |
+| **size**          | `'xsmall' \| 'small' \| 'medium' \| 'large'`                     | `'small'`   | Specifies the size of the rendered Chips withing the ChipGroup                                                                 | ❌       |
+| **value**         | `string \|string[]`                                              | `undefined` | Acts as a controlled component by specifying the ChipGroup value                                                               | ❌       |
+| **intent**        | `'positive' \| 'negative' \| 'neutral'`                          | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                  | ❌       |
+
+## Examples
+
+Here are a few illustrative examples showcasing the utilization of the `Chip` and `ChipGroup` components:
+
+### Basic/Uncontrolled Usage
+
+Single Chip selection:
+
+```jsx
+<Box>
+  <Text> Select Business type: </Text>
```

Comment:
Please add a `label` prop similar to other input components. Using `Text` for label here would disrupt the UI consistency. The use-case of Chip we've seen was used alongside other Input components.

<img width="500" alt="Screenshot 2023-08-22 at 4 00 56 PM" src="https://github.com/razorpay/blade/assets/29686866/c9cb5b3c-6997-4bbe-83a5-adaf96d0a9e3">

Reply:
Right, matching the form UI would need additional efforts. Eg. you likely need a margin bottom additionally to the `Text` which should match what `label` already applies to other input fields.
Also, how would this affect a11y of forms if we don't have a label?

Reply:
For Ally, we have added an `accessibiilityLabel` prop. We are solving labels of various sizes probably by introducing a separate `Label` component for all input components. This development is currently in the pipeline.

Reply:
If we treat the chip as a form element, it should come with a label. Should we introduce an 'accessibilityLabel' as a distinct prop, instead of using a 'label' prop? Once the new Label component is finalised, we should update the chip component accordingly.

Reply:
We have already added an `accessibilityLabel` prop, as the name suggests it's for A11y purposes only and will be visually hidden in the UI.

Reply:
@snitin315 what is the reason behind not keeping the Label prop ?